src/index.js

```diff
@@ -12,4 +21,15 @@ menuButton.addEventListener('click', () => {
     nav.classList.remove('active');
   }
   menuButton.blur(); // Remove focus from button
-});
\ No newline at end of file
+});
+// desktop menu
+const homeLogo = document.querySelector('.logoImg');
+homeLogo.src = logo;
+const dropdownMenu = document.querySelector('.menu_dropdown_arrow');
+dropdownMenu.src = arrowDown;
+const housicon = document.querySelector('.house_icon');
+housicon.src = houseIcon;
+const pencilIcon = document.querySelector('.pencil-square');
+const shielCheck = document.querySelector('.shield-check');
+pencilIcon.src = pencilSquare;
+shielCheck.src = shieldCheck;
```

Comment:
<img width="1345" alt="best-real-state" src="https://github.com/Nelitaa/EasyHouse/assets/98427585/630be657-cfe5-4f38-a658-2bc591b9b4e4">
I've noticed that the images in a particular section of the code aren't displaying as expected. Please investigate and resolve this issue to ensure the images appear correctly in the specified section.